source/docs/example-projects/halloween-treasure-hunt.rst

```diff
@@ -0,0 +1,251 @@
+=======================
+Halloween Treasure Hunt
+=======================
+
+.. contents:: Table of Contents
+    :local:
+
+Introduction
+============
+
+Welcome back to the **Denizen-for-Bukkit Docs**. This is our first example project, so we'll start with the basics. 
```

Comment:
`Welcome back to the **Denizen-for-Bukkit Docs**. ` Odd intro line

Reply:
It's generally not a good idea to have "Welcome back" lines in a text tutorial. In a video, maybe it's appropriate (if only for interactivity and hey-look-we're-not-total-strangers). For the Introduction section, you might want to consider briefly discussing the basic ideas the user should have covered to understand the section. Of course, I've not yet completed all of the sections, so it's perfectly acceptable to add `**[TODO: Add section references]**` as a placeholder or similar, provided that the section(s) required are not written yet.